website/src/components/canvas/Sidebar.jsx

```diff
@@ -104,16 +138,34 @@ class Sidebar extends React.Component {
       <div
       onMouseEnter={this.mouseEnter}
       onMouseLeave={this.mouseLeave}>
-
+       <IconButton
+        iconStyle={styles.arrowOpen}
+        tooltip="Show sidebar"
+        tooltipPosition="bottom-right"
+        onClick={this.onClickOpen}
+        style={{opacity: this.state.closed}}>
+        {this.state.closed ? 0 : 1}
+        <ArrowForward/>
+      </IconButton>
         <Drawer containerStyle={styles.sidebar}
-                open= {true}
+                open= {this.state.isOpen}
                 docked={true}
                 openSecondary={false}
                 style={{opacity: this.state.opacity}}
                 zDepth={0}>
           <div
           style={styles.propertiesSpacing}>
           <ul>
```

Comment:
I wasn't talking about the arrows, but the positioning of the button when expanded. On hamburger menus like [this](http://www.material-ui.com/#/) the button tends to disappear when expanded rather than kind of floating above the rest of the content. 
I've also seen collapsibles like [this](http://bootsnipp.com/snippets/featured/fancy-sidebar-navigation) where the button kind of stays fixed to the outside of the sidebar.